### PR TITLE
add my signature and Chinese translation

### PR DESCRIPTION
**Pull request by TomorJM**

### CODE REVIEW

**Comment by TomorJM** [comment]
~

**Review by 1995re — CHANGES_REQUESTED** [review]
index.ʐh.md was better, but much better if named with index.ch.md

**Comment by TomorJM** [comment]
> index.ʐh.md was better, but much better if named with index.ch.md

good idea

**Comment by jukefr** [comment]
you can also add it to `_config.yml` to make it show on the Github Pages release (rms-open-letter.github.io/)

**Comment by TomorJM** [comment]
> you can also add it to `_config.yml` to make it show on the Github Pages release (rms-open-letter.github.io/)

done

File: index.ch.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+_2021-03-23_
+
+Richard M. Stallman，经常被称为RMS，长期以来一直是自由软件社区的一股危险力量。他已经证明了自己是一个厌恶女性、残疾主义者和跨性别者，以及其他一些不当行为的严重指控。这些信仰在自由软件、数字权利和技术社区中没有立足之地。随着他最近重新回到自由软件基金会董事会，我们呼吁FSF的整个董事会下台，并要求RMS从所有的领导职位上被移除。
```
Comment (AsterNighT) [inline index.ch.md:3]: > 厌恶女性、残疾主义者和跨性别者
Are you translating this using google translate or something? That's not the right words at all. In fact they have opposite meaning.

[the remark this answers is not in the document]
Comment (AsterNighT) [reply index.ch.md:3]:
```suggestion
Richard M. Stallman，或者被简称为RMS（下称RMS），长期以来一直是自由软件社区的一股危险力量。他的不当言行受到广泛抨击，其中包括厌恶女性，歧视残疾人与跨性别者。这些歧视在自由软件、数字权利和技术社区中没有立足之地。随着他最近重新回到自由软件基金会（Free Software foundation，下称FSF）董事会，我们呼吁FSF的整个董事会下台，并要求取消RMS所有的领导职位。
```

File: index.ch.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+作为签署人的我们，相信数字自主的必要性，以及用户自由在保护我们的基本人权方面发挥的强大作用。为了实现软件自由使一切成为可能的承诺，必须在社区内部进行彻底的改变。我们相信，在现在和未来，所有的技术都将赋予人们力量，而不是压迫人们。我们知道，只有在一个技术建立在最基本的层面上尊重我们权利的世界里，这才有可能。虽然Richard M. Stallman以某种形式推广了这些观点，但他并不能代表我们。我们不能宽恕他的行为和观点。我们不承认他或自由软件基金会今天的领导地位。
```
Comment (AsterNighT) [inline index.ch.md:5]:
```suggestion
作为签署人的我们，相信数字自主的必要性，以及用户自由在保护我们的基本人权方面发挥的强大作用。为了实现“软件自由使一切成为可能”的承诺，社区内部必须进行彻底的改变。我们相信，不论现在或是未来，所有的技术都应当赋予人们力量，而不是成为压迫人们的手段。我们知道，只有在一个技术从根本上尊重我们权利的世界里，这才是有可能实现的目标。虽然Richard M. Stallman以某种形式推广了这些观点，但他并不能代表我们。我们不能宽恕他的行为和观点。我们不承认他或FSF今天的领导地位。
```

File: index.ch.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+人们对RMS令人厌恶的思想和行为已经有了足够的容忍。我们不能继续让一个人毁掉我们工作的意义。我们的社区没有容纳理查德·m·斯托曼(Richard M. Stallman)这样的人的空间，我们不会继续忍受他的行为，让他担任领导角色，或以其他方式接受他和他的有害和危险的意识形态。
```
Comment (AsterNighT) [inline index.ch.md:7]:
```suggestion
人们已经受够了RMS令人厌恶的思想和行为。我们不能继续让一个人毁掉我们工作的意义。我们的社区容不下RMS这样的人；我们不会继续忍受他的行为，让他担任领导角色，或以其他方式接受他和他的有害和危险的意识形态。
```

File: index.ch.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+**我们呼吁移除自由软件基金会的整个董事会。** 这些人多年来一直在使用和授权RMS。他们通过允许他重新加入FSF董事会再次证明了这一点。是时候让RMS从自由软件、技术伦理、数字权利和技术社区后退一步了，因为他不能提供我们需要的领导力。 **我们也要求Richard M. Stallman从所有的领导职位上被移除，包括GNU项目。**
```
Comment (AsterNighT) [inline index.ch.md:9]:
```suggestion
**我们呼吁解散自由软件基金会的整个董事会。** 这些人多年来一直在赋予RMS权力。允许他重新加入FSF董事会再次证明了这一点。是时候让RMS从自由软件、技术伦理、数字权利和技术社区退场了，因为他不能提供我们需要的领导。 **我们也要求取消RMS包括GNU项目在内的所有的领导职位。**
```

File: index.ch.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+我们敦促那些有这种立场的人停止支持自由软件基金会。拒绝对与FSF和RMS有关的项目作出贡献。不要在FSF的活动或欢迎RMS和他的不宽容的活动上发言或参加。我们要求自由软件项目的贡献者反对他们项目中的偏见和仇恨。在做这些事情的同时，告诉这些社区和FSF这么做的理由。
```
Comment (AsterNighT) [inline index.ch.md:11]:
```suggestion
我们敦促那些与我们持相同立场立场的人停止支持FSF。拒绝向与FSF和RMS有关的项目作出贡献。不要参加FSF组织的，或者是欢迎RMS的活动，更不要在这些活动上发言。我们要求自由软件项目的贡献者反对他们项目中的偏见和仇恨。在做这些事情的同时，告诉这些社区和FSF这么做的理由。
```

File: index.ch.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+[我们已经详细介绍了几起RMS行为的公开事件][1]。 我们中的一些人有自己的关于RMS的故事，以及我们与他的互动，这些都不是通过电子邮件或视频捕捉到的。我们希望你能读到这些被分享的内容，并考虑到他对我们社区和其他人所做的伤害。
```
Comment (AsterNighT) [inline index.ch.md:13]:
```suggestion
[我们已经详细介绍了几起RMS的不当行为引起的公开事件][1]。 我们中的一些人有自己的关于RMS的故事，以及我们与他的互动，这些事没有通过电子邮件或视频记录。我们希望你能读到这些被分享的内容，并仔细考虑他对我们的社区和其他人所造成的伤害。
```

**Review by AsterNighT — CHANGES_REQUESTED** [review]
I'm not signing this. I'm simply interested in translating.

**Comment by TomorJM** [comment]
@AsterNighT I have updated the optimized translation, please check

File: index.ch.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+_2021-03-23_
+
+Richard M. Stallman(常被称为RMS)，很长时间以来在自由软件社区内都是一股危险的力量。他过去表现出了很多不当行为，比如歧视女性、歧视残障人士、歧视跨性别人群。这些行为不应该在自由软件、数字权利和技术社区中立足。随着他最近重新回到自由软件基金董事会，我们呼吁FSF董事会下台，并要求撤销RMS的所有领导职位。
```
Comment (AsterNighT) [inline index.ch.md:3]:
```suggestion
Richard M. Stallman(常被称为RMS)，很长时间以来在自由软件社区内都是一股危险的力量。他过去表现出了很多不当行为，比如歧视女性、残障人士与跨性别人群。这些行为不应该在自由软件、数字权利和技术社区中存在。他最近重新回到了自由软件基金董事会，我们呼吁FSF董事会下台，并要求撤销RMS的所有领导职位。
```

Comment (AsterNighT) [reply index.ch.md:3]: Try to read your words aloud, if you write awkward sentences, you'll feel twice as awkward reading it.

File: index.ch.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+我们，作为本文件的署名人，相信数字自治的必要性，以及用户自由在保护我们的基本人权方面发挥着关键作用。为了实现软件自由使一切成为可能的承诺，必须在社区内部进行彻底的改变。我们相信，在现在和未来，所有的技术都将赋予人们力量，而不是压迫人们。我们知道，只有在一个技术建立在我们最基本的人权受到尊重的世界里，这才有可能。虽然Richard M. Stallman以某种形式推广了这些观点，但他并不能代表我们。我们不能宽恕他的行为和观点。我们不承认他或自由软件基金会今天的领导地位。
```
Comment (AsterNighT) [inline index.ch.md:5]:
```suggestion
我们，作为本文件的署名人，相信数字自治的必要性，以及用户自由在保护我们的基本人权方面发挥着的关键作用。为了实现“软件自由使一切成为可能”的承诺，必须在社区内部进行彻底的改变。我们相信，在现在和未来，所有的技术都将赋予人们力量，而不是压迫人们。我们知道，只有在一个技术从根本上尊重人权的世界里，这才有可能。虽然RMS以某种形式推广了这些观点，但他并不能代表我们。我们不能宽恕他的行为和观点。我们不承认他或自由软件基金会今天的领导地位。
```

Comment (AsterNighT) [reply index.ch.md:5]: > We know that this is only possible in a world where technology is built to pay respect to our rights at its most foundational levels.

I suggest `at its most foundational levels` is a phrase to modify `technology`, not `our rights`.

And I think you should stick to RMS or Richard M. Stallman, either is ok, but they are not mixed together.

File: index.ch.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+大家已经受够了RMS那些令人极度反感的思想和行为。我们不能继续让一个人毁掉我们工作的意义。我们的社区无法忍受像Richard M.Stallman这样的人存在，我们不会继续忍受他的行为，让他担任领导角色或以其他方式接受他和他的那些有害与危险的思想。
+
+**我们呼吁废除自由软件基金会的整个董事会。** 这些人多年来一直在纵容并赋予RMS权利。比如说，他们允许RMS重新回到FSF董事会。现在是时候让RMS退出自由软件、技术道德、数字权利和技术社区了，因为他提供不了我们需要的领导力。 **我们也要求Richard M. Stallman从所有的领导职位上撤职，包括GNU项目。**
```
Comment (AsterNighT) [inline index.ch.md:9]:
```suggestion
**我们呼吁废除自由软件基金会的整个董事会。** 这些人多年来一直在纵容并赋予RMS权利。比如说，他们允许RMS重新回到FSF董事会。现在是时候让RMS退出自由软件、技术道德、数字权利和技术社区了，因为他提供不了我们需要的领导。 **我们也要求Richard M. Stallman从所有的领导职位上被撤职，包括GNU项目。**
```

Comment (AsterNighT) [reply index.ch.md:9]: I don't think translate `leadership` into `领导力` is a good one. `领导力` refers to one's ability more than a feeling of unity and community harmony.

**Review by AsterNighT — CHANGES_REQUESTED** [review]

**Comment by neilmcgovern** [comment]
Looks like there's still discussion over this, so I'll mark this as draft until it's ready. Just tag us when you're done :)

File: README.md [hunk 10]
```diff
@@ -25,6 +25,7 @@ These translations have been contributed by our community translators. Thanks!
 - [Russian][ru]
 - [Dutch][nl]
 - [Persian][fa]
+- [Chinese][ch]
```
Comment (ehashman) [inline README.md:28]: I have added Traditional Chinese in 31b95e82f408b4a7117b8362184229eb6dacec75.

```suggestion
- [Simplified Chinese][ch]
```

File: _config.yml [hunk 11]
```diff
@@ -20,3 +20,5 @@ languages:
     code: "tr"
   - name: "Russian"
     code: "ru"
+  - name: "Chinese"
```
Comment (ehashman) [inline _config.yml:23]:
```suggestion
  - name: "Simplified Chinese"
```

File: _config.yml [hunk 12]
```diff
@@ -20,3 +20,5 @@ languages:
...
+    code: "ch"
```
Comment (ehashman) [inline _config.yml:24]: I added Traditional Chinese as zh so it's possibly ok to leave this as ch. Or we can fix it so they are differentiated as `zh_CN` and `zh_TW`?

**Review by ehashman — COMMENTED** [review]
Please mark as ready for review when comments addressed!

File: _config.yml (hunk 12 above)
Comment (TomorJM) [reply _config.yml:24]: Yes, you can submit a new PR for revision

**Comment by ehashman** [comment]
I will merge this as is and then do the cosmetic file renames :+1: cheers!

**Pull request merged**